components/automate-cli/pkg/verifyserver/server/api/v1/nfsmount.go

```diff
@@ -0,0 +1,25 @@
+package v1
+
+import (
+	"net/http"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/response"
+	"github.com/gofiber/fiber"
+)
+
+func (h *Handler) NFSMount(c *fiber.Ctx) {
+	reqBody := models.NFSMountRequest{}
+	if err := c.BodyParser(&reqBody); err != nil {
+		h.Logger.Error(err.Error())
+		c.JSON(response.BuildFailedResponse(&fiber.Error{Code: http.StatusBadRequest, Message: "Invalid Body Request"}))
```

Comment:
This will send 200 only,
use like : `c.Next(&fiber.Error{Code: http.StatusBadRequest, Message: "Invalid Body Request"})`